tt_metal/hw/inc/blackhole/noc/noc_parameters.h

```diff
@@ -2,360 +2,29 @@
 //
 // SPDX-License-Identifier: Apache-2.0
 
-#ifndef _NOC_PARAMETERS_H_
-#define _NOC_PARAMETERS_H_
+#include "third_party/umd/src/firmware/riscv/blackhole/noc/noc_parameters.h"
```

Comment:
we should fix this eventually, sigh, wrong for hw to be including from umd...

Reply:
Yes, unfortunately if I revert it back, I get duplicate define errors in the compilation. So I think the umd include is still used somewhere else in the compilation, and will need to be cleaned out for all devices.